axelor-base/src/main/java/com/axelor/apps/base/service/imports/importer/ImporterCSV.java

```diff
@@ -41,7 +47,17 @@ protected ImportHistory process(String bind, String data, Map<String, Object> im
   }
 
   @Override
-  protected ImportHistory process(String bind, String data) throws IOException {
+  protected ImportHistory process(String bind, String data) throws IOException, AxelorException {
     return process(bind, data, null);
   }
+
+  @Override
+  public void checkEntryFilesType(File bind, File data) throws AxelorException {
+    super.checkEntryFilesType(bind, data);
```

Comment:
isn't https://github.com/axelor/axelor-open-suite/pull/12079/files#diff-89d8c85fe40ed1ea54f2a73edc97577105114ae2ebf923ca62f57622d390d986R103 enough? idem for `ImportXml`